### ISSUE TITLE
bug in dbBegin?

### ISSUE THREAD

**Issue by beanumber** [rstats-db/RMySQL#128]
I will feel like a moron if this is me, but I can't see what I'm doing wrong. 
```
> library(DBI)
> con <- dbConnect(RMySQL::MySQL(), group = "rs-dbi")
> dbListTables(con)
character(0)
```
But I can't write the table
```
> dbWriteTable(con, "mtcars", mtcars)
 Show Traceback
 
 Rerun with Debug
 Error in dbFetch(rs, n = -1, ...) : 
  corrupt resultSet, missing fieldDescription

7 dbFetch(rs, n = -1, ...) 
6 dbFetch(rs, n = -1, ...) at DBConnection.R#138
5 dbGetQuery(conn, sql) at DBConnection.R#129
4 dbGetQuery(conn, sql) 
3 .local(conn, name, value, ...) 
2 dbWriteTable(con, "mtcars", mtcars) at DBConnection.R#235
1 dbWriteTable(con, "mtcars", mtcars) 
```
but it *did* create the table
```
> dbListTables(con)
[1] "mtcars"
```
except there is nothing in there
```
> dbReadTable(con, "mtcars")
 [1] mpg  cyl  disp hp   drat wt   qsec vs   am   gear carb
<0 rows> (or 0-length row.names)
```
Note that this also throws the same error:
```
dbBegin(con)
```

I am using MySQL 5.5.46 on Ubuntu.
```
> sessionInfo()
R version 3.2.3 (2015-12-10)
Platform: x86_64-pc-linux-gnu (64-bit)
Running under: Ubuntu 14.04.3 LTS

locale:
 [1] LC_CTYPE=en_US.UTF-8       LC_NUMERIC=C               LC_TIME=en_US.UTF-8       
 [4] LC_COLLATE=en_US.UTF-8     LC_MONETARY=en_US.UTF-8    LC_MESSAGES=en_US.UTF-8   
 [7] LC_PAPER=en_US.UTF-8       LC_NAME=C                  LC_ADDRESS=C              
[10] LC_TELEPHONE=C             LC_MEASUREMENT=en_US.UTF-8 LC_IDENTIFICATION=C       

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] DBI_0.3.1.9008

loaded via a namespace (and not attached):
[1] magrittr_1.5  tools_3.2.3   Rcpp_0.12.2   stringi_1.0-1 RMySQL_0.10.7 stringr_1.0.0
```

**Comment by beanumber** [comment]
FYI, it works on Ubuntu 15.04 with MySQL 5.6.27. Is this a version issue?

**Comment by krlmlr** [comment]
Could you please try the dev version of RMySQL? `devtools::install_github("rstats-db/RMySQL")`

**Comment by beanumber** [comment]
OK, that worked. Although the `dbConnect()` now wants an explicit password, even though it is already in `.my.cnf`. 

Sorry!

**Comment by krlmlr** [comment]
The password issue should be resolved in #90. Can you confirm?

**Comment by beanumber** [comment]
Yes, that worked. Thanks! <going back to being a moron/>
